### ISSUE TITLE
warning: constant ::Fixnum is deprecated at Ruby 2.4.1

### ISSUE THREAD

**Issue by eugene-nikolaev** [macournoyer/thin#338]
Occured at:

```
ruby 2.4.1p111 (2017-03-22 revision 58053) [x86_64-linux]
```

```
~/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/thin-1.7.0/lib/thin/server.rb:107: warning: constant ::Fixnum is deprecated
```

**Comment by voikya** [comment]
This appears to have been fixed in `master` for several months, but I'd appreciate it if a new gem release could be published with the fix.

**Comment by MatzFan** [comment]
This is fixed in current release (1.7.2). Issue should be closed.
